app/services/message_formatter.rb

```diff
@@ -24,6 +25,16 @@ def self.format_message(message)
     MessageFormatter.new(message).format
   end
 
+  def self.extract_mentions(message)
+    MessageFormatter.new(message).mentions
```

Comment:
If we initialize the method each time we call an static method, why won't use a singleton instead?